website/meta/universe.json

```diff
@@ -4062,6 +4062,33 @@
             "author_links": {
                 "github": "yasufumy"
             }
+        },
+        {
+            "id": "spacy-pythainlp",
+            "title": "spaCy-PyThaiNLP",
+            "slogan": "PyThaiNLP for spaCy",
+            "description": "This package wraps the PyThaiNLP library to add support for Thai to spaCy.",
+            "github": "PyThaiNLP/spaCy-PyThaiNLP",
+            "code_example": [
+                "import spacy",
+                "import spacy_pythainlp.core",
+                "",
+                "nlp = spacy.blank('th')",
+                "nlp.add_pipe('pythainlp')",
+                "doc = nlp('ผมเป็นคนไทย   แต่มะลิอยากไปโรงเรียนส่วนผมจะไปไหน  ผมอยากไปเที่ยว')",
+                "",
+                "print(list(list(doc.sents)))",
+                "# output: [ผมเป็นคนไทย   แต่มะลิอยากไปโรงเรียนส่วนผมจะไปไหน  , ผมอยากไปเที่ยว]"
+            ],
+            "code_language": "python",
+            "author": "Wannaphong Phatthiyaphaibun",
+            "author_links": {
+                "twitter": "@wannaphong_p",
+                "github": "wannaphong",
+                "website": "https://github.com/PyThaiNLP/spaCy-PyThaiNLP"
```

Comment:
Note that because this is in the author section, it shouldn't be the repo URL, but a personal page. I think https://iam.wannaphong.com/ would be a good fit, or you could omit it.

Reply:
> Note that because this is in the author section, it shouldn't be the repo URL, but a personal page. I think https://iam.wannaphong.com/ would be a good fit, or you could omit it.

Thank you! I added my personal website.